actix-session/src/storage/session_key.rs

```diff
@@ -17,8 +17,8 @@ use derive_more::{Display, From};
 /// let session_key: Result<SessionKey, _> = key.try_into();
 /// assert!(session_key.is_err());
 /// ```
-#[derive(Debug, PartialEq, Eq)]
-pub struct SessionKey(String);
+#[derive(Debug, PartialEq, Eq, Default, Clone)]
+pub struct SessionKey(pub String);
```

Comment:
You don't want to expose the inner field here. 
We can change `SessionKey` to be `SessionKey(secrecy::Secret<String>)` and expose `into_inner`/`inner` methods to access the inner value.

Reply:
In the code we are impl a `AsRef`
https://github.com/actix/actix-extras/blob/d2b7229f88c5b27c6c8310621e26910fcbcecbfc/actix-session/src/storage/session_key.rs#L38

```
method `as_ref` has an incompatible type for trait
expected fn pointer `fn(&storage::session_key::SessionKey) -> &str`
   found fn pointer `fn(&storage::session_key::SessionKey) -> &secrecy::Secret<std::string::String>`
```

However it's not clear to me how to do with with secrecy data
https://docs.rs/secrecy/0.8.0/secrecy/struct.Secret.html#trait-implementations

Should I be looking for a trait AsRef here?

If I update to
```rust
impl AsRef<secrecy::Secret<String>> for SessionKey {
    fn as_ref(&self) -> &secrecy::Secret<String> {
        &self.0
    }
}
```
Does this mean I need to update redis_rs.rs / redis_actor.rs / cookie.rs